### DISCUSSION TITLE
如何查看我的收割機有沒有正常運作？

### DISCUSSION THREAD

**Discussion by tgc1122** [Chia-Network/chia-blockchain#7872] [Q&A]
我有3收割機為什麼space pool裡才看到2？有時看到3？那我怎樣知道到底有沒有正常運作？

**Comment by keliew** [comment]
check its `debug.log`
https://github.com/Chia-Network/chia-blockchain/wiki/How-to-Check-If-Everything-is-Working-(or-Not)
